docs/guide/metamask-extension-provider.md

```diff
@@ -0,0 +1,55 @@
+# MetaMask Extension Provider
+
+A module for providing a MetaMask [provider](https://github.com/ethereum/wiki/wiki/JavaScript-API#web3currentprovider) to other WebExtensions.
```

Comment:
I think I will adjust the language so the module is documented from its own perspective.
```suggestion
A module for accessing the user's MetaMask [provider](https://github.com/ethereum/wiki/wiki/JavaScript-API#web3currentprovider) from other WebExtensions.
```